lib/views/bidding_slack_view.ex

```diff
@@ -5,25 +5,25 @@ defmodule Aprb.Views.BiddingSlackView do
     artwork_data = fetch_sale_artwork(event["lotId"])
     %{
       text: ":gavel: #{event["type"]} on #{artwork_data[:permalink]}",
-      attachments: "[{
-                      \"fields\": [
-                        {
-                          \"title\": \"Amount\",
-                          \"value\": \"#{format_price(event["amountCents"] / 100)}\",
-                          \"short\": true
+      attachments: [%{
+                      fields: [
+                        %{
```

Comment:
I love that replacing the string with maps is basically one-to-one with less noise.